src/AzSK.ADO/Framework/Configurations/SVT/ADO/ADO.AgentPool.json

```diff
@@ -162,7 +162,23 @@
         "AutomatedFix"
       ],
       "Enabled": true
+    },
+    {
+      "ControlID": "ADO_AgentPool_Dont_Grant_Broader_Group_Access_As_Approvers",
+      "Description": "Broader groups (contributors, project valid users, etc.) should not be added as approvers on agent pool.",
+      "Id": "AgentPool230",
+      "ControlSeverity": "High",
+      "Automated": "Yes",
+      "MethodName": "CheckBroaderGroupApproversOnAgentPool",
+      "Rationale": " Any user/ group can be added as an approver to the agent pool, which gives users the permission to approve any run of pipelines accessing the resource even if they do not have access to the agent pool. To prevent illegitimate consumption and approval of the resource, ensure that broader groups are not added as approvers.",
```

Comment:
Any user/group can be added as an approver to the agent pool, which gives user/group the permission to approve any run of pipelines accessing the resource even if they do not have access to the agent pool. To prevent illegitimate consumption and approval of the resource, ensure that broader groups are not added as approvers.